pages/common/k3s.md

```diff
@@ -0,0 +1,36 @@
+# k3s
+
+> Install and manage lightweight Kubernetes clusters using the k3s CLI.
+> More information: <https://docs.k3s.io/cli/k3s>.
+
+- Install a single-node K3s server with default settings:
+
+`curl {{[-sfL|--silent --fail --location]}} https://get.k3s.io | sh -s - server`
+
+- Install a K3s server without deploying a local agent:
+
+`curl {{[-sfL|--silent --fail --location]}} https://get.k3s.io | sh -s - server --disable {{agent}}`
+
+- Install a K3s agent that connects to a server:
+
+`curl {{[-sfL|--silent --fail --location]}} https://get.k3s.io | sh -s - agent {{[-s|--server]}} {{https://myserver:6443}} {{[-t|--token]}} {{mytoken}}`
```

Comment:
I'm not really liking curling random things off the internet and the running them.